sentinel-core/slot/base/stats_slot.h

```diff
@@ -10,24 +10,24 @@ namespace Slot {
 class StatsSlot : public Slot {
  public:
   virtual ~StatsSlot() = default;
+
   /*
    * Statistics class slot default always continue the rest of the slot
    * TODO(tianqian.zyf): TokenResultSharedPtr should be passed over the entire
    * slot chain using the SlotChainContext method.
    */
-  bool IsContinue(const TokenResultSharedPtr& token) override {
-    last_token_result_ = token;
+  bool IsContinue(const TokenResultSharedPtr& token,
+                  const EntryContextSharedPtr& context) override {
+    if (context != nullptr) {
```

Comment:
`assert(context)? `Why the context equal nullptr?

Reply:
It should not be `null` logically. Just in case of unexpected errors. Can `assert(context)` achieve this?